components/organisms/Layout.js

```diff
@@ -20,6 +21,20 @@ export const Layout = ({
   return (
     <div className="overflow-x-hidden">
       <header>
+        {/* layout for the Early Access Alpha banner start */}
+        <div className="bg-gray-100">
+          <div className="layout-container ">
+            <AlphaBanner
+              alphaBannerText={t("alphaBannerText")}
+              alphaBannerDesc={t("alphaBannerDesc")}
+              feedbackButton={`/benefit/`}
```

Comment:
I think this was meant to bring you to the alpha page

Reply:
it's just a banner